scripts/fetch-validation-data.py

```diff
@@ -0,0 +1,50 @@
+import os
```

Comment:
I'd love a docstring at the top of those two scripts, with a high level explanation of what they're doing.